GYBITG/Repositories/UsersRepository.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  UsersRepository.swift
+//  GYBITG
+//
+//  Created by Sharief Youssef on 5/28/19.
+//
+
+import Foundation
+
+//The protocol for the VideoView
+protocol UsersRepositoryProtocol{
+    //to store all videos
+    func addUser(userToAdd: User) -> String
+    
+}
+
+class UsersRepository: UsersRepositoryProtocol{
+  
+    internal var users = [User]()
```

Comment:
See James' comment about using the AppDelegate to retain the list of users across ViewControllers